lib/js/_utils/__mocks__/_get-latest-version.js

```diff
@@ -12,19 +12,14 @@
  * @param {String} packageName name of package to check
  * @return {Promise<String>}
  */
-function getLatestVersion(name) {
-	return new Promise((resolve, reject) => {
-		process.nextTick(
-			() => {
-				if ((name === 'global-error')) {
-					reject(new Error('error thrown'));
-				}
-				if ((name === 'global-unpublished')) {
-					resolve(null);
-				}
-				resolve('2.0.0');
-			});
-	});
+async function getLatestVersion(name) {
+	if ((name === 'global-error')) {
```

Comment:
do we not need to `process.nextTick()` ?

Reply:
the associated tests can be written:
```
	test('The data is a valid version number', async () => {
		expect.assertions(1);
		expect(await getLatestVersion('valid')).toBe('1.0.0');
	});
```

instead of the current:
```
	test('The data is a valid version number', async () => {
		expect.assertions(1);
		await getLatestVersion('valid').then(data => {
			expect(data).toBe('1.0.0');
		});
	});
```